patches/src/main/resources/youtube/settings/host/values/arrays.xml

```diff
@@ -37,48 +37,56 @@
         <item>LARGE_FORM_FACTOR_WIDTH_DP</item>
     </string-array>
     <string-array name="revanced_change_start_page_entries">
+        <item>@string/revanced_change_start_page_entry_browse</item>
+        <item>@string/revanced_change_start_page_entry_courses</item>
         <item>@string/revanced_change_start_page_entry_default</item>
-        <item>@string/revanced_change_start_page_entry_search</item>
-        <item>@string/revanced_change_start_page_entry_shorts</item>
-        <item>@string/revanced_change_start_page_entry_subscriptions</item>
         <item>@string/revanced_change_start_page_entry_explore</item>
-        <item>@string/revanced_change_start_page_entry_notifications</item>
+        <item>@string/revanced_change_start_page_entry_fashion</item>
+        <item>@string/revanced_change_start_page_entry_gaming</item>
+        <item>@string/revanced_change_start_page_entry_history</item>
         <item>@string/revanced_change_start_page_entry_library</item>
-        <item>@string/revanced_change_start_page_entry_your_clips</item>
         <item>@string/revanced_change_start_page_entry_liked_videos</item>
-        <item>@string/revanced_change_start_page_entry_watch_later</item>
-        <item>@string/revanced_change_start_page_entry_history</item>
-        <item>@string/revanced_change_start_page_entry_trending</item>
-        <item>@string/revanced_change_start_page_entry_courses</item>
-        <item>@string/revanced_change_start_page_entry_gaming</item>
         <item>@string/revanced_change_start_page_entry_live</item>
-        <item>@string/revanced_change_start_page_entry_music</item>
         <item>@string/revanced_change_start_page_entry_movies</item>
+        <item>@string/revanced_change_start_page_entry_music</item>
+        <item>@string/revanced_change_start_page_entry_news</item>
+        <item>@string/revanced_change_start_page_entry_notifications</item>
+        <item>@string/revanced_change_start_page_entry_podcasts</item>
+        <item>@string/revanced_change_start_page_entry_search</item>
+        <item>@string/revanced_change_start_page_entry_shopping</item>
+        <item>@string/revanced_change_start_page_entry_shorts</item>
         <item>@string/revanced_change_start_page_entry_sports</item>
-        <item>@string/revanced_change_start_page_entry_browse</item>
+        <item>@string/revanced_change_start_page_entry_subscriptions</item>
+        <item>@string/revanced_change_start_page_entry_trending</item>
+        <item>@string/revanced_change_start_page_entry_watch_later</item>
+        <item>@string/revanced_change_start_page_entry_your_clips</item>
     </string-array>
     <string-array name="revanced_change_start_page_entry_values">
         <item>ORIGINAL</item>
```

Comment:
The order of `revanced_change_start_page_entries` and `revanced_change_start_page_entry_values` must match.

Otherwise, the labels and settings will not match, and incorrect settings will be saved.